lib/modules/measure/MeasureService.ts

```diff
@@ -444,3 +448,61 @@ export class MeasureService {
     return measureName.asset;
   }
 }
+
+/**
+ * Create a new document in the collection asset-history, for each asset states
+ */
+function historizeAssetStates(
```

Comment:
This method should be async so it will always return a promise instead of throw an exception.

Reply:
Oupsy !
Done :)